wb_vandalism/features/feature.py

```diff
@@ -26,3 +27,14 @@ def __init__(self, property):
 
     def _process(self, changed_claims):
         return self.property in [claims[0].id for claims in changed_claims]
+
+
+class has_in_comment(Modifier):
+    def __init__(self, check):
+        self.check = check
+        name = "has_in_comment({0})".format(repr(check))
+        super().__init__(name, self._process, returns=bool,
+                         depends_on=[comment])
+
+    def _process(self, comment):
+        return self.check in comment
```

Comment:
What do you think of making this a regex check?  That would give us some more flexibility.  It looks like all the strings specified in features/diff would are intended to match the beginning of the comment string, but they would also match if they appeared elsewhere in the comment.